### DISCUSSION TITLE
Question: How to setup in existing project

### DISCUSSION THREAD

**Discussion by LiorBanai** [Dotnet-Boxed/Templates#713] [Q&A]
### Question

Is there a tutorial on how to set up NuGet Package Boxed on an existing repo/project?

### Project template

NuGet Package Boxed

## Environment

VS 16.6.4
Windows 10

**Comment by VictorioBerra** [comment]
These are templates. It's like opening visual studio, and clicking file -> new project. What comes out is a bunch of pre-built files and code. You couldn't just do that inside of an existing project. It would dump all the files inside. There's no way for the templates to see there's already a file like .csproj and then merge with it.

If you want all the bells and whistles of this template, generate it, then copy over your code files.

**Comment by LiorBanai** [comment]
@VictorioBerra  Thanks. 
I have created a new project and used that as baseline for my needs (in this case, automatic publish to nuget.org)
